### PR TITLE
Feature/cxec 17126

### PR DESCRIPTION
**Pull request by steinsebastian**

### CODE REVIEW

File: feature-libs/product-configurator/rulebased/components/tab-bar/configurator-tab-bar.component.ts [hunk 1]
```diff
@@ -44,6 +52,37 @@ export class ConfiguratorTabBarComponent {
     )
   );
 
+  getTabIndexConfigTab(): number {
```
Comment (ChristophHi) [inline feature-libs/product-configurator/rulebased/components/tab-bar/configurator-tab-bar.component.ts:60]: Please add JS doc for new public methods

File: feature-libs/product-configurator/rulebased/components/tab-bar/configurator-tab-bar.component.ts [hunk 2]
```diff
@@ -44,6 +52,37 @@ export class ConfiguratorTabBarComponent {
...
+      if (!isOvPage) {
+        tabIndex = -1;
+      }
+    });
+    return tabIndex;
+  }
+
+  switchTabOnArrowPress(event: KeyboardEvent, currentTab: string): void {
+    if (event.code === 'ArrowLeft' || event.code === 'ArrowRight') {
+      event.preventDefault();
```
Comment (ChristophHi) [inline feature-libs/product-configurator/rulebased/components/tab-bar/configurator-tab-bar.component.ts:92]: Why is that needed? I can see no change in behaviour if that is omitted (or is there a default handling for ArrowLeft and ArrowRight I am not aware of?)

**Review by ChristophHi — CHANGES_REQUESTED** [review]

File: feature-libs/product-configurator/rulebased/components/tab-bar/configurator-tab-bar.component.ts (hunk 1 above)
Comment (steinsebastian) [reply feature-libs/product-configurator/rulebased/components/tab-bar/configurator-tab-bar.component.ts:60]: done

File: feature-libs/product-configurator/rulebased/components/tab-bar/configurator-tab-bar.component.ts (hunk 2 above)
Comment (steinsebastian) [reply feature-libs/product-configurator/rulebased/components/tab-bar/configurator-tab-bar.component.ts:92]: It is necessary to restrict the keypress to ArrowLeft and ArrowRight.
If it is omitted then any keypress will lead to focus change between the tabs (e.g. if you press any letter-key like "g", etc.).
But here only ArrowLeft and ArrowRight should trigger the focus change.

**Review by ChristophHi — APPROVED** [review]

**Pull request merged**